test/unit/domain/fido-credential.test.ts

```diff
@@ -31,7 +31,7 @@ import { validate, unpackAttestationObject, packAttestationObject, FIDOAttestati
 
 describe('fido-credential', () => {
   describe('validate', () => {
-    const credential: tpAPI.Schemas.FIDOPublicKeyCredential = {
+    const credential: tpAPI.Schemas.FIDOPublicKeyCredentialAttestation = {
```

Comment:
pretty sure this should be `Assertion` but `fido-credential` is going to be deleted in my PR anyway.

Reply:
oh nvm. excuse me.